src/types/geometry.rs

```diff
@@ -40,3 +40,69 @@ impl ToJson for Geometry {
         }
     }
 }
+
+impl Geometry {
+    pub fn from_json(json_geometry: &Object) -> Geometry {
+        match json_geometry.get("type").unwrap().as_string().unwrap() {
+            "Point" => Geometry::Point(Point::from_json(json_geometry)),
+            "MultiPoint" => Geometry::MultiPoint(MultiPoint::from_json(json_geometry)),
+            "LineString" => Geometry::LineString(LineString::from_json(json_geometry)),
+            "MultiLineString" => Geometry::MultiLineString(MultiLineString::from_json(json_geometry)),
+            "Polygon" => Geometry::Polygon(Polygon::from_json(json_geometry)),
+            "MultiPolygon" => Geometry::MultiPolygon(MultiPolygon::from_json(json_geometry)),
+            "GeometryCollection" => Geometry::GeometryCollection(GeometryCollection::from_json(json_geometry)),
+            _ => panic!(),
```

Comment:
It's reachable, if the input string is invalid GeoJSON. There are several other points in the code where it panics because of invalid input. I'll clean them up in the next PR.